tools/releaseBuild/azureDevOps/templates/release-ReleaseToNuGet.yml

```diff
@@ -8,20 +8,21 @@ steps:
 
 - pwsh: |
     #Exclude all global tool packages. Their names start with 'PowerShell.'
-    Copy-Item "$ENV:PIPELINE_WORKSPACE/releasePipeline/finalResults/*.nupkg" -Destination "$(Pipeline.Workspace)/release" -Exclude "PowerShell.*.nupkg" -Force
+    $null = New-Item -ItemType Directory -Path "$(Pipeline.Workspace)/release"
+    Copy-Item "$ENV:PIPELINE_WORKSPACE/releasePipeline/finalResults/*.nupkg" -Destination "$(Pipeline.Workspace)/release" -Exclude "PowerShell.*.nupkg" -Force -Verbose
```

Comment:
Seeing our inclusion of `-Verbose` here, makes me think it would be really nice if we just had a variable we could set to make the whole build spit out verbose PowerShell output. But it does also make sense as a default

Reply:
Getting all verbose can be too much sometimes. Specially with `import-module` etc.

Reply:
Its more for logging what gets copied to where.